signalrkore/src/commonMain/kotlin/eu/lepicekmichal/signalrkore/HubConnection.kt

```diff
@@ -400,12 +406,51 @@ class HubConnection private constructor(
         }
     }
 
+    private suspend fun processReceivedInvocation(message: HubMessage.Invocation) {
+        if (message is HubMessage.Invocation.Blocking && message.invocationId != null && subscribersWithResult[message.target] != true) {
+            logger.log(Logger.Level.ERROR, "Failed to find a value returning handler for ${message.target} method. Sending error to server.")
```

Comment:
In the Java library this is logged as a warning, which make sense to me. I think it would be nice if the `Level` enum could be extended with the warning.